python/cherrypy/server.py

```diff
@@ -0,0 +1,19 @@
+import cherrypy
+
+class Root(object):
```

Comment:
What is the spec for the server? Maybe it's better/cleaner to use a method dispatcher instead of traversal.